packages/contents/src/tokens.ts

```diff
@@ -129,10 +129,14 @@ export interface IContents {
  * Commonly-used mimetypes
  */
 export namespace MIME {
-  export const JS = 'application/javascript';
```

Comment:
This mimetype is deprecated, see "Legacy JavaScript mimetypes" in https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types. `text/javascript` replaces it

Reply:
Maybe we still need to keep it around for now?

Uploading and opening a `.js` file seems to be giving the following with this PR:

![image](https://user-images.githubusercontent.com/591645/173312533-2523cbd9-99e9-4f8f-8d5a-c52b27de2415.png)

While the file is correctly handled on `main`: https://jupyterlite.readthedocs.io/en/latest/_static/lab/

![image](https://user-images.githubusercontent.com/591645/173312667-ba7d8489-13c9-4dbe-abdb-07817e0dbe68.png)

Reply:
Thanks for the review, I reverted the change then.